client/core/core.go

```diff
@@ -3111,6 +3114,23 @@ func (c *Core) Login(pw []byte) (*LoginResult, error) {
 				if err != nil {
 					c.log.Errorf("Unable to connect to %s wallet (start and sync wallets BEFORE starting dex!): %v",
 						unbip(wallet.AssetID), err)
+					// NOTE: Details for this topic is in the context of fee
+					// payment, but the subject pertains to a failure to connect
+					// to the wallet. This is better than just an ERR log.
+					subject, _ := c.formatDetails(TopicWalletConnectionWarning)
+					c.notify(newWalletConfigNote(TopicWalletConnectionWarning, subject, err.Error(),
+						db.ErrorLevel, wallet.state()))
```

Comment:
`Login` was not communicating wallet connect errors except on the console as ERR logs, and that's one reason for the confusion in the issue this PR resolves.  Now you get a report of the connect error in the browser ntfns

![image](https://user-images.githubusercontent.com/9373513/149846168-cd92e58d-ae30-4f8b-afd0-4ab185a68c96.png)


2022-01-17 17:01:46.426 [INF] CORE: Connecting wallet for dcr
2022-01-17 17:01:46.426 [INF] CORE: Connecting wallet for btc
2022-01-17 17:01:46.427 [ERR] CORE: Unable to connect to dcr wallet (start and sync wallets BEFORE starting dex!): connectWallet: dcrwallet connect error: dial tcp 127.0.0.1:19572: connect: connection refused
2022-01-17 17:01:46.430 [ERR] CORE: notify: |ERROR| (walletconfig) Wallet connection warning - connectWallet: dcrwallet connect error: dial tcp 127.0.0.1:19572: connect: connection refused

Reply:
Note that this bit was already merged into release-v0.4 in https://github.com/decred/dcrdex/pull/1424